twitchAPI/types.py

```diff
@@ -278,8 +278,9 @@ class ChatEvent(Enum):
     CHAT_CLEARED = 'chat_cleared'
     """Triggered when a user was banned, timed out or all messaged from a user where deleted"""
     WHISPER = 'whisper'
-    """Triggered when someone whispers to your bot. NOTE: You need the :const:`~twitchAPI.types.AuthScope.WHISPERS_READ` Auth Scope 
-    to get this Event."""
+    """Triggered when someone whispers to your bot. NOTE: You need the :const:`~twitchAPI.types.AuthScope.WHISPERS_READ` Auth Scope to get this Event."""
```

Comment:
can you revert the line break change here? it was added to keep within the 150 character length